cypress/e2e/functional/admin-portal/file-deletion-spec.cy.js

```diff
@@ -139,7 +139,64 @@ describe('Admin - Groups screen', () => {
     it('should reject same marked user deletion', () => {
       cy.get('.govuk-table__row').eq(2).find('.govuk-button.govuk-button--secondary').click();
 
-      cy.get('.govuk-heading-l').should('contain', 'You cannot delete this transcript');
+      cy.get('.govuk-heading-l').should('contain', 'You cannot delete this transcript file');
+    });
+
+    it('should validate required input', () => {
+      cy.get('.govuk-table__row').eq(1).find('.govuk-button.govuk-button--secondary').click();
+
+      cy.get('#confirm-button').click();
+
+      cy.get('.govuk-error-message').should('contain', 'Select your decision');
+      cy.get('.govuk-error-summary__list').should('contain', 'Select your decision');
+    });
+
+    it('should delete file', () => {
+      cy.get('.govuk-table__row').eq(1).find('.govuk-button.govuk-button--secondary').click();
+
+      cy.get('.govuk-heading-xl').should('contain', 'Delete transcript file');
+
+      cy.get('.govuk-table__header').should('have.length', 8);
+      cy.get('.govuk-table__header').eq(0).should('contain', 'Transcript ID');
+      cy.get('.govuk-table__header').eq(1).should('contain', 'Case ID');
+      cy.get('.govuk-table__header').eq(2).should('contain', 'Hearing date');
+      cy.get('.govuk-table__header').eq(3).should('contain', 'Courthouse');
+      cy.get('.govuk-table__header').eq(4).should('contain', 'Courtroom');
```

Comment:
Courtroom isn't a column on the figma and they are also in the wrong order

<img width="898" alt="Screenshot 2024-10-03 at 16 19 56" src="https://github.com/user-attachments/assets/e21c6989-8268-4013-ad19-70678bead239">

Reply:
Columns update for delete transcript file view